src/ReactiveUI.Wpf/ReactiveUI.Wpf.csproj

```diff
@@ -14,7 +14,7 @@
     <WarningLevel>4</WarningLevel>
     <TargetFrameworkProfile />
     <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
-    <PackageId>ReactiveUI.WPF</PackageId>    
+    <PackageId>ReactiveUI.WPF</PackageId>
```

Comment:
These whitespace changes just seemed to happen, didn't think it was worth fighting, but can do if we think it is.

Reply:
Nah don't worry about it.